test/sql/parser/test_columns_unpacked.test

```diff
@@ -0,0 +1,75 @@
+# name: test/sql/parser/test_columns_unpacked.test
+# group: [parser]
+
+query I
+select COALESCE(*COLUMNS(*)) from (select NULL, 2, 3) t(a, b, c);
+----
+2
+
+query I
+select column_name from (describe select COALESCE(*COLUMNS(*)) from (select NULL, 2, 3) t(a, b, c));
+----
+COALESCE(t.a, t.b, t.c)
+
+# Aliasing the result
+query I
+select column_name from (describe select COALESCE(*COLUMNS(*)) as a from (select NULL, 2, 3) t(a, b, c));
+----
+a
+
+statement ok
+create table contains_test as select '123abc234' a, 4 b, 'abc' c;
+
+# Feed columns a and c into contains, equivalent to contains(a, c)
+query I
+select contains(*COLUMNS('[a|c]')) from contains_test;
+----
+true
+
+# COLUMNS and *COLUMNS are equivalent as a root expression
+query II
+select COLUMNS('[a|c]') from contains_test;
+----
+123abc234	abc
+
+query II
+select *COLUMNS('[a|c]') from contains_test;
```

Comment:
Let's maybe support `*COLUMNS` only inside expressions, i.e. make this throw an error?